framework/platform/platform.cpp

```diff
@@ -116,6 +120,16 @@ ExitCode Platform::initialize(const std::vector<Plugin *> &plugins)
 		return ExitCode::Close;
 	}
 
+	if (!app_requested())
+	{
+		LOGE("An app was not requested, can not continue");
+		for (auto &help : parser->help())
+		{
+			LOGI(help);
+		}
+		return ExitCode::Help;
```

Comment:
With this change, the help text is written twice. Once here, and once in `Platform::terminate`.
That is, you should either `LOGI(help)` here or return `ExitCode::Help`, but not both.
Probably the same holds on lines 83-87.

Reply:
Ah nice spot!